frontend/projects/upgrade/src/app/core/segments/segments.service.spec.ts

```diff
@@ -11,6 +11,7 @@ const MockStateStore$ = new BehaviorSubject({});
 describe('SegmentService', () => {
   let mockStore: any;
   let segmentsDataService: any;
+  let localStorageService: any;
```

Comment:
any -> LocalStorageService

Reply:
To mock we use any in all testcases in frontend and backend, can't give specific type like Localstorage as we aren't going to created all properties for `localStorageService` in `segmentService` testcase.